scripts/neighbor_advertiser

```diff
@@ -236,42 +236,22 @@ def get_vlan_addresses(vlan_interface):
     ipv6_addr = []
     ipv6_prefix = []
 
-    '''
-    Sample output for "ip addr show Vlan101"
-
-	218: Vlan101@Bridge: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
-		link/ether 98:5d:82:01:d8:48 brd ff:ff:ff:ff:ff:ff
-		inet 10.171.22.113/29 scope global Vlan101
-		   valid_lft forever preferred_lft forever
-		inet6 2603:10b0:10f:843c::1/64 scope global
-		   valid_lft forever preferred_lft forever
-		inet6 fe80::9a5d:82ff:fe01:d848/64 scope link
-		   valid_lft forever preferred_lft forever
-    '''
-    try:
-        out = subprocess.check_output(['ip', 'addr', 'show', vlan_interface])
-        for line in out.splitlines():
-            keys=line.strip().split(' ')
-            if keys[0] == 'inet':
-                ipv4_addr.append(keys[1].split('/')[0])
-                ipv4_prefix.append(keys[1].split('/')[1])
-            elif keys[0] == 'inet6':
-                ipv6_addr.append(keys[1].split('/')[0])
-                ipv6_prefix.append(keys[1].split('/')[1])
-            elif keys[0] == 'link/ether':
-                mac_addr = keys[1]
-    except Exception:
-        log.log_error('failed to get %s addresses from o.s.' % vlan_interface)
+    vlan_intfs = config_db.get_table('VLAN_INTERFACE')
+    for item in vlan_intfs.keys():
+        if vlan_interface in item[0]:
+            ipaddr = IPNetwork(item[1])
```

Comment:
there would be cases where there is no item[1], so we would have to skip those. Like just this "VLAN_INTERFACE|Vlan2000". I would suggest to just use `get_vlan_addr ` function as it already handles that case